include/boost/beast/_experimental/test/impl/stream.hpp

```diff
@@ -77,6 +77,16 @@ class basic_stream<Executor>::read_op : public detail::stream_read_op_base
           return net::get_associated_allocator(h_);
         }
 
+        using cancellation_slot_type =
+            net::associated_cancellation_slot_t<Handler>;
+
+        cancellation_slot_type
+        get_cancellation_slot() const noexcept
```

Comment:
What if people are currently using `websocket::stream` with a NextLayer that doesn't support cancelation slots? Will they get a compile error about a missing nested type and `get_cancellation_slot` function?

Reply:
No, it will just get ignored. It's a property on the handler, not the stream